README.md

```diff
@@ -13,6 +13,7 @@
 - [🔗 Celo Subgraph Playground](https://thegraph.com/hosted-service/subgraph/bulla-network/bulla-contracts-celo)
 - [🔗 Avalanche Subgraph Playground](https://thegraph.com/hosted-service/subgraph/bulla-network/bulla-contracts-avalanche)
 - [🔗 Goerli Subgraph Playground](https://thegraph.com/hosted-service/subgraph/bulla-network/bulla-contracts-goerli)
+- [🔗 Base Testnet Subgraph Playground](https://api.studio.thegraph.com/query/13828/bulla-contracts-base/v1)
```

Comment:
Should this be called "Based Testnet Subgraph Playground"?

Reply:
great catch :)